### PR TITLE
added entry for SaysWho

### PR DESCRIPTION
**Pull request by afriedman412**
## Checklist

- [ ] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Review by adrianeboyd — COMMENTED** [review]
Thanks for this contribution! I was able to install and run the examples following the instructions in the README without any problems.

I'd recommend specifying that this package (currently) only supports English.

I'd also recommend using the most recent coreference model:

https://github.com/explosion/spacy-experimental/releases/download/v0.6.1/en_coreference_web_trf-3.4.0a2-py3-none-any.whl

Error messages that explain how to install missing models might be helpful, especially for the experimental coreference model, since that might not be obvious to many users.

And in the universe entry it would be nice to have a short code example and a bit more detail about how this package relates to spacy.

File: website/meta/universe.json [hunk 1]
```diff
@@ -16,6 +16,22 @@
             },
             "category": ["extension"],
             "tags": []
+        },
+                {
+            "id": "sayswho",
+            "title": "SaysWho",
+            "slogan": "Quote identification, attribution and resolution.",
+            "description": "A Python package for identifying and attributing quotes in text. It uses a combination of logic and grammer to find quotes and their speakers, then uses a coreferencing model to better clarify who is speaking.",
```
[the remark this answers is not in the document]
Comment (afriedman412) [reply website/meta/universe.json:24]: wow this is embarassing lol

**Comment by afriedman412** [comment]
is there a way to install the coreferencing model without automatically installing SpaCy 3.4?

**Comment by adrianeboyd** [comment]
You can install the model with `--no-deps`, but then it doesn't pull in `spacy-experimental`. It basically still is all on the experimental side and maybe not quite ready to be used in another package like this, although it's still really nice to see what people are doing with the coref model!

I think that this particular model should currently be compatible with any newer versions of spacy, but I haven't tested it and you'll see warnings.

Is there a particular reason you need to use a newer version of spacy?

**Comment by afriedman412** [comment]
ok noted -- yeah it works fine with the newer versions as far as I can tell so you might as well use the latest version if possible, you know?

File: website/meta/universe.json [hunk 2]
```diff
@@ -16,6 +16,31 @@
             },
             "category": ["extension"],
             "tags": []
+        },
+                {
+            "id": "sayswho",
+            "title": "SaysWho",
+            "slogan": "Quote identification, attribution and resolution",
+            "description": "A Python package for identifying and attributing quotes in text. It uses a combination of SpaCy functionality, logic and grammar to find quotes and their speakers, then uses the SpaCy coreferencing model to better clarify who is speaking. Currently English only.",
```
Comment (adrianeboyd) [inline website/meta/universe.json:24]:
```suggestion
       {
            "id": "sayswho",
            "title": "SaysWho",
            "slogan": "Quote identification, attribution and resolution",
            "description": "A Python package for identifying and attributing quotes in text. It uses a combination of spaCy functionality, logic and grammar to find quotes and their speakers, then uses the spaCy coreferencing model to better clarify who is speaking. Currently English only.",
```

**Comment by adrianeboyd** [comment]
Thanks for the updates!

File: website/meta/universe.json [hunk 3]
```diff
@@ -17,6 +17,31 @@
             "category": ["extension"],
             "tags": []
         },
+       {
```
Comment (adrianeboyd) [inline website/meta/universe.json:20]:
```suggestion
        {
```

**Pull request merged**